public/addEdit.js

```diff
@@ -0,0 +1,144 @@
+import { enableInput, inputEnabled, message, setDiv, token } from "./index.js";
+import { showJobs } from "./jobs.js";
+
+let addEditDiv = null;
+let company = null;
+let position = null;
+let status = null;
+let addingJob = null;
+
+
+export const handleAddEdit = () => {
+  addEditDiv = document.getElementById("edit-job");
+  company = document.getElementById("company");
+  position = document.getElementById("position");
+  status = document.getElementById("status");
+  addingJob = document.getElementById("adding-job");
```

Comment:
I agree with you and I have const at the beginning. However, when I'm trying to change let to const, I'm getting an error in console "addEdit.js:12 Uncaught TypeError: Assignment to constant variable." When I search why do I have some bumps here, I figured out I've declared addEditDiv, company, position, status, and addingJob as const and set their values to null. Later in the handleAddEdit function, I'm trying to assign new values to these variables, which is not allowed.   So, can I use "let" here?

Reply:
The error above says line 12 is the culprit and not reassigning the value. Please check your code again, as I’m pretty sure you can keep the HTML node variable as const since you’re just changing the node value.